docs/getting_started/benchmark_owner_demo.md

```diff
@@ -74,124 +74,122 @@ medperf test run \
+For the Reference Model MLCube, the submission should include:
 
-    - The submission should include:
+- The URL to the hosted mlcube manifest file:
 
-        a. The URL to the hosted mlcube manifest file:
+    ```text
+    {{ model_mlcube }}
+    ```
 
-        ```text
-        {{ model_mlcube }}
-        ```
+- The URL to the hosted mlcube parameters file:
 
-        b. The URL to the hosted mlcube parameters file:
+    ```text
+    {{ model_params }}
+    ```
 
-        ```text
-        {{ model_params }}
-        ```
+- The URL to the hosted additional files tarball file:
 
-        c. The URL to the hosted additional files tarball file:
+    ```text
+    {{ model_add }}
+    ```
 
-        ```text
-        {{ model_add }}
-        ```
+Use the following command to submit:
 
-    - Command to submit:
+    ```bash
+    medperf mlcube submit \
+    --name my-modelref-cube \
+    --mlcube-file "{{ model_mlcube }}" \
+    --parameters-file "{{ model_params }}" \
+    --additional-file "{{ model_add }}"
+    ```
 
-        ```bash
-        medperf mlcube submit \
-        --name my-modelref-cube \
-        --mlcube-file "{{ model_mlcube }}" \
-        --parameters-file "{{ model_params }}" \
-        --additional-file "{{ model_add }}"
-        ```
+#### Metrics MLCube:
 
-3. The Metrics MLCube:
+For the Metrics MLCube, the submission should include:
 
-    - The submission should include:
+- The URL to the hosted mlcube manifest file:
 
-        a. The URL to the hosted mlcube manifest file:
+    ```text
+    {{ metrics_mlcube }}
+    ```
 
-        ```text
-        {{ metrics_mlcube }}
-        ```
+- The URL to the hosted mlcube parameters file:
 
-        b. The URL to the hosted mlcube parameters file:
+    ```text
+    {{ metrics_params }}
+    ```
 
-        ```text
-        {{ metrics_params }}
-        ```
+Use the following command to submit:
 
-    - Command to submit:
+    ```bash
+    medperf mlcube submit \
+    --name my-metrics-cube \
+    --mlcube-file "{{ metrics_mlcube }}" \
+    --parameters-file "{{ metrics_params }}" \
+    ```
 
-        ```bash
-        medperf mlcube submit \
-        --name my-metrics-cube \
-        --mlcube-file "{{ metrics_mlcube }}" \
-        --parameters-file "{{ metrics_params }}" \
-        ```
-
-Each of the three MLCubes will be assigned by a server UID. You can check them by running:
+Each of the three MLCubes will be assigned by a server UID. You can check the server UIDs for each MLCube by running:
 
 ```bash
 medperf mlcube ls --mine
 ```
 
-Next, we will learn how to host the demo dataset.
+Next, you will learn how to host the demo dataset.
 
-## 5. Hosting the Demo Dataset
+## 5. Host the Demo Dataset
 
-The demo dataset should be packaged in a specific way as a compressed tarball file. Looking at the folder stucture in the workspace:
+The demo dataset should be packaged in a specific way as a compressed tarball file. The folder stucture in the workspace should look like the following:
```

Comment:
The demo dataset should be packaged in a specific way as a compressed tarball file. The folder stucture in the workspace `currently looks` like the following: